libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -184,6 +184,21 @@
         LOGIN_INSERTED_INVALID_URL,
         LOGIN_AUTOFILL_CREDENTIALS_FILLED,
         LOGIN_AUTOFILL_CREDENTIALS_UPDATED,
+        LOGIN_PROLOGUE_VIEWED,
```

Comment:
For the new items, I wanted to match the tracks naming convention which is Context, Source, Action. So in this case the context is the Login feature, the source is the screen being viewed, and the action is viewed.
Having the keys match the naming convention should make it easier when reviewing analytics in Tracks. 
It also occurs to me just now there's no need to wait to wire these keys up in the trackers (the keys in the iOS app are in a separate lib so waiting was necessary). 
Lets put this PR on hold and I'll add that change as well.  Then there will be less to do when adding the calls to the updated login screens.